contrib/clojure-package/examples/bert/get_bert_data.sh

```diff
@@ -19,11 +19,14 @@
 
 set -e
 
-data_path=model
+data_path=data
 
 if [ ! -d "$data_path" ]; then
   mkdir -p "$data_path"
   curl https://s3.us-east-2.amazonaws.com/mxnet-scala/scala-example-ci/BertQA/vocab.json -o $data_path/vocab.json
   curl https://s3.us-east-2.amazonaws.com/mxnet-scala/scala-example-ci/BertQA/static_bert_qa-0002.params -o $data_path/static_bert_qa-0002.params
   curl https://s3.us-east-2.amazonaws.com/mxnet-scala/scala-example-ci/BertQA/static_bert_qa-symbol.json -o $data_path/static_bert_qa-symbol.json
+  curl https://media.githubusercontent.com/media/gigasquid/mxnet-data/master/static_bert_base_net-symbol.json -o $data_path/static_bert_base_net-symbol.json
+  curl https://media.githubusercontent.com/media/gigasquid/mxnet-data/master/static_bert_base_net-0000.params -o $data_path/static_bert_base_net-0000.params
```

Comment:
@lanking520 Would it be possible to host the BERT Base params on on the same bucket as the other `static-bert-qa` params too rather than my github?

Reply:
@gigasquid I just added them to the bucket for you. You can now find them with the equivalent paths: 
```
https://s3.us-east-2.amazonaws.com/mxnet-scala/scala-example-ci/BertQA/static_bert_base_net-symbol.json
https://s3.us-east-2.amazonaws.com/mxnet-scala/scala-example-ci/BertQA/static_bert_base_net-0000.params
```

Reply:
thank you!!